### ISSUE TITLE
Campfire shouldn't damage you through vehicles

### ISSUE THREAD

**Issue by Haox09** [deferred-impact/battledudes-issue-tracker#215]
When I was writing #214 , I thought of this. Don't let campfires deal damage to you when you drive over them. Period.

**Comment by deferred-impact** [comment]
Meh. Couldn't care less.

**Comment by Haox09** [comment]
broo💀💀💀💀💀

**Comment by ghost** [comment]
Yeah..I agree. It's pointless, and the damage is too little. Besides, you could simply drive around them, if you choose to drive over it, then the damage is your fault.
